google-cloud-examples/src/main/java/com/google/cloud/examples/bigquery/snippets/BigQuerySnippets.java

```diff
@@ -571,22 +572,18 @@ public boolean cancelJobFromId(String jobName) {
   /**
    * Example of running a query.
    */
-  // [TARGET query(QueryRequest)]
+  // [TARGET query(QueryJobConfiguration, QueryOption...)]
   // [VARIABLE "SELECT unique(corpus) FROM [bigquery-public-data:samples.shakespeare]"]
   public QueryResponse runQuery(String query) throws InterruptedException {
     // [START runQuery]
-    QueryRequest request = QueryRequest.of(query);
-    QueryResponse response = bigquery.query(request);
-    // Wait for things to finish
-    while (!response.jobCompleted()) {
-      Thread.sleep(1000);
-      response = bigquery.getQueryResults(response.getJobId());
-    }
+    QueryJobConfiguration queryConfig =
+        QueryJobConfiguration.newBuilder(query).setUseLegacySql(true).build();
```

Comment:
Ideally the runQuery example would show a standard SQL sample and we'd have a separate sample for explicitly setting to legacy SQL.

Reply:
I agree, but I believe this should be made as a separate PR. The goal of this is to modify the bigquery client itself (changing snippets only if they are broken).